### PR TITLE
Add a check result entry to nested forms

### PR DESCRIPTION
**Pull request by DAGpro**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

**Review by arogachev — CHANGES_REQUESTED** [review]
The tests are needed. What's the purpose of these changes exactly?

**Comment by samdark** [comment]
@DAGpro so it calls nested object's hooks, right?

**Comment by DAGpro** [comment]
@samdark right

**Review by vjik — CHANGES_REQUESTED** [review]
1) Add test with check error path in nested object.
2) Add line to changelog

**Review by vjik — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 ## 2.1.1 under development
 
-- no changes in this release.
+- Bug #748: Fix hook processing for nested objects that implement `PostValidationHookInterface` when `Nested` rule with
+  rules is used (@DAGpro)
```
Comment (arogachev) [inline CHANGELOG.md:6]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
 ## 2.1.1 under development
 
-- no changes in this release.
+- Bug #748: Fix hook processing for nested objects that implement `PostValidationHookInterface` when `Nested` rule with
```
Comment (arogachev) [inline CHANGELOG.md:5]:
```suggestion
- Bug #748: Fix hook processing for nested objects that implement `PostValidationHookInterface` when `Nested` rule is used (@DAGpro)
```

**Review by arogachev — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 ## 2.1.1 under development
 
-- no changes in this release.
+- Bug #748: Fix hook processing for nested objects that implement `PostValidationHookInterface` when `Nested` rule is used (@DAGpro)
```
Comment (vjik) [inline CHANGELOG.md:5]:
```suggestion
- Bug #748: Fix hook processing for nested objects that implement `PostValidationHookInterface` when `Nested` rule with
  specified rules is used (@DAGpro)
```

**Comment by vjik** [comment]
@DAGpro thanks!

**Pull request merged**